test/unit/spec/VerifyPIV_spec.js

```diff
@@ -0,0 +1,134 @@
+/* eslint max-params: [2, 18], max-statements: 0 */
```

Comment:
nit: this file can be `PIVLogin_spec.js` or maybe `LoginWithPIV_spec.js`

Reply:
Actually would it be ok to keep this name? Just because it is in sync with VerifyPIVController.js, and follows the pattern for other spec files